openshift/system/Dockerfile

```diff
@@ -91,11 +82,33 @@ RUN export ${BUNDLER_ENV} >/dev/null \
     && rm log/*.log \
     && chmod g+w /opt/system/config
 
-USER 1001
 ADD openshift/system/entrypoint.sh /opt/system/entrypoint.sh
+USER 1001
 EXPOSE 3000 9306
 # TODO: dumb-init!
 ENTRYPOINT ["/opt/system/entrypoint.sh"]
 CMD ["unicorn", "-c", "config/unicorn.rb", "-E", "${RAILS_ENV}", "config.ru"]
 
-# vim: set ft=dockerfile:
+
+FROM porta-base AS porta-sphinx-config
+ENV THINKING_SPHINX_ADDRESS=0.0.0.0 \
+    THINKING_SPHINX_PORT=9306 \
+    THINKING_SPHINX_PID_FILE=/var/run/sphinx/searchd.pid \
+    THINKING_SPHINX_BATCH_SIZE=100 \
+    THINKING_SPHINX_INDICES_LOCATION=/var/lib/sphinx \
+    THINKING_SPHINX_CONFIGURATION_FILE=/opt/system/config/standalone.sphinx.conf \
+    RAILS_ENV=production \
+    SECRET_KEY_BASE=dummy \
+    DATABASE_URL='mysql2://root:@localhost/porta'
```

Comment:
database name doesn't matter because this layer is scrapped. Port is not desirable because we expect connection ove unix socket, what MySQL does with `localhost` host.